the_role_specs/specs/factories/user.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :user, class: User do
-    email   { Faker::Internet.email }
+    sequence(:email) { |n| "email#{n}@example.com" }
```

Comment:
created for avoid of randomized faker. They can have generate email which can't validate